docs/testing.rst

```diff
@@ -81,3 +81,77 @@ When the reason for the breackage is identified there are two options to go forw
     * If the failure comes from an intended change in behaviour adjust Anaconda or the tests
     * If the failure uncovers an actual regression, file bugs for the given Fedora components. If it does not look like the issue will be
       fixed quickly, work around in Anaconda or add a `naughty override file <https://github.com/cockpit-project/bots/tree/main/naughty/>`_, thus marking the expected failure pattern.
+
+Fix Web UI pixel tests
+----------------------
+
+So pixel tests fail on your PR - now what ?
+
+There are essentially two options:
+
+    * the test uncovered a in your PR bug that needs to be fixed in the code
```

Comment:
I would change to "uncovered a bug in your PR that".

Reply:
Oops, fixed, thanks! :)